addons/website_sale/views/variant_templates.xml

```diff
@@ -25,9 +25,13 @@
                             t-att-name="'ptal-%s' % ptal.id">
                             <t t-foreach="ptavs" t-as="ptav">
                                 <option t-att-value="ptav.id"
+                                    t-attf-class="js_variant_change #{ptal.attribute_id.create_variant}"
                                     t-att-data-value_id="ptav.id"
                                     t-att-data-value_name="ptav.name"
                                     t-att-data-attribute_name="attribute.name"
+                                    t-att-data-attribute_id="ptav.attribute_id.id"
+                                    t-att-data-attribute_value_id="ptav.product_attribute_value_id.id"
+                                    t-att-data-sequence="ptav.product_attribute_value_id.sequence"
```

Comment:
Why using snake case instead of camel case for the dataset ?
```suggestion
                                    t-att-data-attribute-id="ptav.attribute_id.id"
                                    t-att-data-attribute-value-id="ptav.product_attribute_value_id.id"
                                    t-att-data-sequence="ptav.product_attribute_value_id.sequence"
```